test/playwright/e2e/filters.spec.ts

```diff
@@ -152,9 +152,9 @@ test('filters are updated when media type changes', async ({ page }) => {
 
   // Only CC0 checkbox is checked, and the filter button label is '1 Filter'
   await assertCheckboxStatus(page, 'cc0')
-  const filterButtonSelector =
-    '[aria-controls="filter-sidebar"], [aria-controls="filter-modal"]'
-  await expect(page.locator(filterButtonSelector)).toHaveText('1 Filter')
+  await expect(
```

Comment:
`aria-controls` doesn't seem to work anymore: I couldn't find any elements with `filter-sidebar` or `filter-modal` ids. If the sidebar and the modal are not present in the DOM at the same time, we could just use a single `filters` id, and simplify this locator to use `[aria-controls="filters"].

Reply:
The `filter-modal` aria-controls doesn't appear to exist anywhere in the codebase on `main`, that's why I removed it from this selector.

Not sure why the `filter-sidebar` one isn't working anymore though!

Reply:
Actually, it seems this must be a regression from a previous PR as I can't find anything with a `filter-sidebar` ID in `main` either.

Reply:
It seems to be a regression from a long time ago, because I couldn't find neither of those ids in the code. But it would surely be nice to fix :)

Reply:
Where are those id's meant to go? On the wrapper elements of the sidebar and the modal?